File: arrays/java/sorted_square.java
import java.util.Arrays;

public class sorted_square {

    public static int[] sortedSquares(int[] nums) {
        int[] result = new int[nums.length];

        for (int i = 0; i < nums.length; i++) {
            result[i] = (int) Math.pow(nums[i], 2);
        }

        Arrays.sort(result);
        return result;
    }

    public static void main(String[] args) {
        System.out.println(Arrays.toString(sortedSquares(new int[]{-4, -1, 0, 3, 10})));
        System.out.println(Arrays.toString(sortedSquares(new int[]{-7, -3, 2, 3, 11})));
    }
}

File: arrays/javascript/sorted-square.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
const sortedSquares = (nums) => {
    const result = [];
    
    for (let value of nums) {
        result.push(Math.pow(value, 2));
    }

    result.sort((a, b) => a - b);

    return result;
};

console.log(sortedSquares([-4, -1, 0, 3, 10]));
console.log(sortedSquares([-7, -3, 2, 3, 11]));